src/screens/AccountManagementScreen.tsx

```diff
@@ -1,7 +1,140 @@
+"use client";
+
+import { useEffect, useState } from "react";
+import { DotsThree, MagnifyingGlass, Plus } from "@phosphor-icons/react";
+import { CgSpinner } from "react-icons/cg";
+import { User, UserType } from "@prisma/client";
+import classNames from "classnames";
+
+function formatUserType(type: UserType): string {
+  return type
+    .toLowerCase()
+    .split("_")
+    .map((word) => word.charAt(0).toUpperCase() + word.slice(1))
+    .join(" ");
+}
+
+const filterMap: Record<string, (user: User) => boolean> = {
+  All: () => true,
+  "Hope for Haiti Staff": (user) => user.type === UserType.STAFF,
+  Partners: (user) => user.type === UserType.PARTNER,
+};
```

Comment:
It might be better to use an enum for the keys of the filterMap for stricter typing on line 26.